src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -495,3 +496,32 @@ The following steps are performed:
 | `--kubeconfig` `<string>`        | Absolute path(s) to the kubeconfig file(s)
 | `--namespace` `<string>`         | Namespace in which Submariner is installed (default "submariner-operator")
 | `--yes`                          | Automatically answer yes to confirmation prompt
+
+### Running `subctl diagnose` from a Pod in cluster
+
+As of release 0.12.0, a new `subctl` image containing the subctl binary is now available in submariner repos. This can be
```

Comment:
I wasnt sure of how we handle documentation for different versions and best practices around it. Had an idea that shouldn't use versions, but best way to get right answer is suggest wrong answer first :)

Reply:
I think documenting the version in which a feature appeared is fine, as long as we avoid references to newness. Especially since we support multiple versions, but only provide a single set of documentation, we can’t expect users to read all the docs with a copy of the release notes for reference; what’s more, the link between a feature in “how to use” documentation and the corresponding entry in the release notes might not be obvious.

Reply:
@dfarrell07 @skitt FWIW, I recently added https://github.com/submariner-io/submariner-website/pull/697 which also mentions a specific version.  I don't have a strong opinion on this, but at least let's try to be consistent in our approach. 

Ideally, we should properly tag the website and publish per-release version of the docs, but until we do so we are going to face this challenge. In fact, we can already tag the website, the challenge is more with the Hugo theme and how it cab be adopted to support clear and easy navigation between versions (see https://github.com/submariner-io/submariner-website/issues/428). Maybe something we should bring in 0.13 planning?

Reply:
> I'll send a PR

https://github.com/submariner-io/submariner-website/pull/705